### DISCUSSION TITLE
bunker unloading position

### DISCUSSION THREAD

**Discussion by alepinci** [Courseplay/courseplay#7468] [Q&A] [answered]
Hi, is it normal that when a driver reach the bunker silo in which he has to reverse unload, he starts unloading as soon as he reach the chaff (the red line)? This way it ends everytime unloading out of the bunker because he starts too soon. 
![immagine](https://user-images.githubusercontent.com/38883600/131557892-0c16dd59-e55d-44d3-9dd2-f525916b2b09.png)

**Comment by Tensuko** [comment, marked as answer]
That is normal and is already adjusted that he drive a bit more into the the chaff before unload.
Won't change that.

**Reply by alepinci** [reply]
Ok clear, thanks for the reply :)
